app/src/ui/publish-repository/publish-repository.tsx

```diff
@@ -64,7 +65,8 @@ export class PublishRepository extends React.Component<
 
   private async fetchOrgs(account: Account) {
     const api = API.fromAccount(account)
-    const orgs = await api.fetchOrgs()
+    const orgs = (await api.fetchOrgs()) as Array<IAPIUser>
```

Comment:
No need for parenthesis here

Reply:
Prettier puts them in:

**Prettier 1.9.2**
[Playground link](https://prettier.io/playground/#N4Igxg9gdgLgprEAuc0DOMAEEBOBzNTAXkwEMB3UgSy1IAcqA6AMzhjAAsB5fNACgCUZQgEEcOUgE8APAEkRABVkBVNHBwA+ADpQQAGhAQ6MKumShS4iOQWWEaZCFIA3CFQAm+pxmTNSAGzUDACMJMABrNgBlOlIwKig8ZBgcAFc4A3cIMF8AoJAEtRwYBQk8AFtSXMCMkAArNAAPACEwyJgo0nK4ABkEuGr8iFSYOhGAJkHa2JwixxhJOjg0MBwqYy86NdgAdQ8YDmQADgAGAy2INR2JOkct5fVnAYMcOABHVKpX0tIKqqQ-DUDGpylQpsCEnh-HAAIqpCDwZJpWowUjBPbuA7IcYGFLUfyQgDCEHKlUcUGgzxAqTUABU0Q4AXk4ABfFlAA)

**Input:**
```tsx
const orgs = await api.fetchOrgs() as Array<IAPIUser>

```

**Output:**
```tsx
const orgs = (await api.fetchOrgs()) as Array<IAPIUser>

```

Reply:
Prettier interferes again. Do you know if there is an option to disable this?

Reply:
No, there’s [a policy](https://github.com/prettier/prettier/issues/40) to strongly resist adding configuration.

Reply:
It's unfortunate really. I'd think they'd be pushing to get rid of syntax that increase the noise to signal ratio.

Reply:
It seems like the parentheses [were added](https://github.com/prettier/prettier/issues/1913) because `yield` wasn’t parsed the same without them, but `await` doesn’t seem to have this limitation. If you’re interested in changing this back, please open an issue on the Prettier repo (I’d open one for you, but I don’t fully understand the reason why the one without parentheses is better).

Some history on these types of styles: originally, Prettier avoided printing parentheses wherever possible, but they started [adding them](https://prettier.io/playground/#N4Igxg9gdgLgprEAuEA3AhgJwAQDMITYC8262AZOdgEbYA+d2YIANCBAA4wCW0AzslBZMEAO4AFLAgEp0qCNwAmrEOj4xkudABs+cNtUzowAazgwAyh2PcoAc2QxMAV30hFEZki263tvZgw4kZ2ALbomjp6bABWfAAeAEJGpuYW6KFwADK2cJG+bBDOMBzFAEz50SDWmAHIIDAAnhxwfGCY3FwqHB2wAOpKMAAWyAAcAAxsPRB6fUYc9T2tcJioeWyYcACOztybwehhEd5Rbnqh3JVntnbacACKzhDwji5uMOjUA4rDyGVsTnQ3G0NwAwhBQuF6lBoOsQM49AAVT4yHx6AC+6KAA) to make code easier to understand.

Reply:
> I don’t fully understand the reason why the one without parentheses is better

It's really the same as `var a = 5` being better than `var a = (5)`.

Both are valid, but the one with the parenthesis increases the noise to signal ratio. The parenthesis doesn't add anything other than more characters to the screen. There is no precedence to worry about here.

Reply:
I understand your point here, but `await foo as bar` seems ambiguous to me — it isn‘t immediately clear if that means `await (foo as bar)` or `(await foo) as bar`.